src/Donations/Endpoints/ListDonations.php

```diff
@@ -208,14 +219,15 @@ public function getTotalDonationsCount(): int
     }
 
     /**
+     * @unreleased Make the class extendable
      * @since 2.24.0 Remove joins as it uses ModelQueryBuilder and change clauses to use attach_meta
      * @since      2.21.0
      *
      * @param QueryBuilder $query
      *
      * @return array{0: QueryBuilder, 1: array<DonationMetaKeys>}
      */
-    private function getWhereConditions(QueryBuilder $query): array
+    protected function getWhereConditions(QueryBuilder $query): array
```

Comment:
I definitely agree that we should make the visibility of this method `protected`.